fenix/app/src/main/java/org/mozilla/fenix/onboarding/AdjustActivatedUserWorker.kt

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.onboarding
+
+import android.content.Context
+import androidx.work.ExistingWorkPolicy
+import androidx.work.OneTimeWorkRequest
+import androidx.work.WorkManager
+import androidx.work.Worker
+import androidx.work.WorkerParameters
+import com.google.android.gms.common.util.VisibleForTesting
+import org.mozilla.fenix.components.metrics.Event
+import org.mozilla.fenix.ext.components
+import org.mozilla.fenix.ext.settings
+import org.mozilla.fenix.utils.Settings
+
+/**
+ * Worker that tracks conditions for "Activated" event for new users.
+ */
+class AdjustActivatedUserWorker(
+    context: Context,
+    workerParameters: WorkerParameters,
+) : Worker(context, workerParameters) {
+
+    override fun doWork(): Result {
+        val settings = applicationContext.settings()
+        val metrics = applicationContext.components.analytics.metrics
+
+        metrics.track(Event.ActivatedUserData.FirstWeekUsageActivity)
+
+        if (shouldSendAdjustActivatedEvent(settings)) {
+            settings.adjustUserActivated = true
+            metrics.track(Event.GrowthData.UserActivated)
+        }
+
+        return Result.success()
+    }
+
+    companion object {
+        private const val DAYS_IN_FIRST_WEEK_ACTIVATED_THRESHOLD = 3
+        private const val SEARCHES_IN_FIRST_WEEK_ACTIVATED_THRESHOLD = 1
+
+        private const val ONBOARDING_ADJUST_WORK_NAME = "org.mozilla.fenix.onboarding.adjust-work"
+
+        /**
+         * Sends Adjust "Activated" Event if needed
+         */
+        fun sendAdjustOnboardingEventIfNeeded(context: Context) {
+            val instanceWorkManager = WorkManager.getInstance(context)
+
+            val adjustOnboardingWork = OneTimeWorkRequest.Builder(AdjustActivatedUserWorker::class.java)
+                .build()
+
+            instanceWorkManager.beginUniqueWork(
+                ONBOARDING_ADJUST_WORK_NAME,
+                ExistingWorkPolicy.KEEP,
+                adjustOnboardingWork,
+            ).enqueue()
+        }
+
+        @VisibleForTesting
+        internal fun shouldSendAdjustActivatedEvent(settings: Settings): Boolean {
+            if (!settings.adjustUserActivatedSent &&
+                settings.adjustUseCount.value >= DAYS_IN_FIRST_WEEK_ACTIVATED_THRESHOLD &&
+                settings.adjustEarlySearchCount.value >= SEARCHES_IN_FIRST_WEEK_ACTIVATED_THRESHOLD
+            ) {
+                return true
+            }
+            return false
```

Comment:
This function looks to check the same conditions that the `Event.GrowthData.UserActivated` branch in `shouldTrack` does. `shouldTrack`will run during `track` anyway, so I think this function can be removed.

Reply:
Removed this file completely (see below comment).